### PR TITLE
YALB-491: Wire news cards and views

### PR DESCRIPTION
**Pull request by ModulesUnraveled**
### Description of work
- Wires news cards, and views

### Functional testing steps:
- [ ] Follow review steps here: https://github.com/yalesites-org/yalesites-project/pull/61

### CODE REVIEW

File: templates/node/node--news--list-item.html.twig [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+{% set date__formatted %}
+  {% include "@atoms/date-time/date-time.twig" with {
+    date_time__start: date_formatted,
+    date_time__format: 'date',
+  } %}
+{% endset %}
+
+{% embed "@molecules/cards/reference-card/reference-card.twig" with {
+  reference_card__overline: date__formatted,
+  reference_card__heading: content.field_teaser_title,
```
Comment (nJim) [inline templates/node/node--news--list-item.html.twig:12]: Let's fall back on the node title if a teaser title is not provided.

File: templates/node/node--news--card.html.twig [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+{% set date__formatted %}
+  {% include "@atoms/date-time/date-time.twig" with {
+    date_time__start: date_formatted,
+    date_time__format: 'date',
+  } %}
+{% endset %}
+
+{% embed "@molecules/cards/reference-card/reference-card.twig" with {
+  reference_card__overline: date__formatted,
+  reference_card__heading: content.field_teaser_title,
```
Comment (nJim) [inline templates/node/node--news--card.html.twig:12]: Let's fall back on the node title if a teaser title is not provided.

**Review by nJim — APPROVED** [review]
Only one change requested. For teaser headings, let's fall back on the node title if the teaser title is not provided. Currently the teaser title is provided as an optional override, but it is likely that most folks will not use this field.

**Review by nJim — CHANGES_REQUESTED** [review]
Only one change requested. For teaser headings, let's fall back on the node title if the teaser title is not provided. Currently the teaser title is provided as an optional override, but it is likely that most folks will not use this field.

File: templates/node/node--news--list-item.html.twig (hunk 1 above)
Comment (ModulesUnraveled) [reply templates/node/node--news--list-item.html.twig]: done

File: templates/node/node--news--card.html.twig (hunk 2 above)
Comment (ModulesUnraveled) [reply templates/node/node--news--card.html.twig]: done

**Pull request merged**